src/components/common/DateInput.svelte

```diff
@@ -43,8 +43,7 @@
 <div class={`container`}>
   <input
     class={`input ${
-      (value === min && moment.utc().format("YYYY-MM-DD") === min) ||
-      (value <= min && moment.utc().format("YYYY-MM-DD") <= min)
+      value < min && moment.utc().format("YYYY-MM-DD") <= min
```

Comment:
I don't believe this is working, but when I went to test it, the calendar is not showing anymore.

Reply:
Brave... but have chrome/firefox for testing.
Good point on input disabled.
Will respond again if I give it a try.

Reply:
Yeah, I also don't see the date picker on chrome
![image](https://user-images.githubusercontent.com/20739535/216058514-f0f9d22d-bf61-42ab-933c-35a97c21641b.png)

Reply:
It seems to work on Firefox and Brave for me through

Reply:
Without the date picker this is very frustrating to use as it's not possible to type the date either. So the only option is to select one of the predefined options below

Reply:
Hm.. I'm using Chrome but it is visible for me

Reply:
Possibly something that was already fixed on main. I just saw you pulled from main and now the date picker is working fine for me  👍

Reply:
Still investigating, but on my end actually Chrome works correctly 😅 

Chrome:
<img width="536" alt="Screenshot 2023-02-02 at 08 02 08" src="https://user-images.githubusercontent.com/13741335/216319511-54ab72a2-2d26-466f-b550-124eced64685.png">

Brave:
<img width="518" alt="Screenshot 2023-02-02 at 08 02 17" src="https://user-images.githubusercontent.com/13741335/216319565-067cbfe9-fc23-4ae0-b460-16ea05530c13.png">

Reply:
Yeah, weird that the min value is different

Reply:
Well, looks like on Brave, the connected account already has a lock, so that changes the min value

Reply:
this was caused by a plugin I had on my machine. No further discussion needed.